cmake/kokkos_arch.cmake

```diff
@@ -378,6 +378,17 @@ IF (KOKKOS_ARCH_HSW)
   )
 ENDIF()
 
+IF (KOKKOS_ARCH_RISC64_rv64imafdcv)
```

Comment:
Why is the suffix in lowercase?  We generally use uppercase for these.
Also you did not actually register a `KOKKOS_ARCH_OPTION()` for it.

Reply:
No, I did not register the KOKKOS_ARCH_OPTION() yet.